spec/dummy/db/schema.rb

```diff
@@ -10,28 +10,29 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_03_04_200630) do
+ActiveRecord::Schema[7.2].define(version: 2019_03_04_200630) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```

Comment:
cloudsql does support this extension but none of our instances have it enabled, fyi

why do we need it? I don't see any new functions being added

Reply:
yo i'm a big fat liar
```
inventory-c=> \dx
                           List of installed extensions
  Name   | Version |   Schema   |                   Description
---------+---------+------------+--------------------------------------------------
 hstore  | 1.8     | public     | data type for storing sets of (key, value) pairs
 plpgsql | 1.0     | pg_catalog | PL/pgSQL procedural language
(2 rows)
```

Reply:
still not sure what part of the changes we need it for though so lemme know cause i'm curious

Reply:
oh
> In Rails 7.0 and later, datetime columns are generated with a default precision of 6 (microseconds) if not explicitly specified.

okay then ... nevermind, I was thinking we could just use a non-date type of column like integer if we're only storing seconds but then

> datetime columns integrate seamlessly with ActiveRecord's time-related functionalities.

so... I guess that's why.